docs/docs-content/clusters/pcg/manage-pcg/pcg-upgrade.md

```diff
@@ -0,0 +1,65 @@
+---
+sidebar_label: "Upgrade a PCG"
+title: "Upgrade a PCG"
+description: "Learn how to upgrade a Private Cloud Gateway (PCG) in Palette."
+hide_table_of_contents: false
+sidebar_position: 40
+tags: ["pcg"]
+---
+
+A PCG upgrade event can be broken down into two categories: Palette agent updates and cluster profile updates. Depending
+on the upgrade event, different steps are required to complete the upgrade.
+
+Palette updates are tied to the Palette platform version. Self-hosted Palette instances must upgrade to the next minor
```

Comment:
```suggestion
Palette agent updates are tied to the Palette platform version. Self-hosted Palette instances must upgrade to the next minor
```
Is this correct? It makes sense for us to discuss the two update versions directly after introducing them so I assume that Palette update here is referring to Palette agent updates. Also, since we have a lot of concepts that sounds similar but are different in this section, it may be best to explicitly say that PCG agent and Palette Agent are the same or just standardize and only use Palette Agent,